modules/transcription-service-amberscript/src/main/java/org/opencastproject/transcription/amberscript/AmberscriptTranscriptionService.java

```diff
@@ -271,16 +291,33 @@ public Job startTranscription(String mpId, Track track, String language, String
       jobtype = getAmberscriptJobType();
     }
 
+    int numberOfSpeakers = defaultNumberOfSpeakers;
+    if (track.getMediaPackage() != null) {
+      Set<String> speakers = new HashSet<>();
+      if (speakerMetadataField.equals(SpeakerMetadataField.presenter)
+          || speakerMetadataField.equals(SpeakerMetadataField.both)) {
+        speakers.addAll(Arrays.asList(track.getMediaPackage().getCreators()));
```

Comment:
Weird mixture between terms. If you count the creators, the setting should also be named creator and not presenter.